ldms/python/ldmsd/ldmsd_controller

```diff
@@ -1500,6 +1500,35 @@ class LdmsdCmdParser(cmd.Cmd):
     def complete_auth_add(self, text, line, begidx, endidx):
         return self.__complete_attr_list('auth_add', text)
 
+    def do_option(self, arg):
+        """
+        ONLY SUPPORTED IN CONFIGURATION FILES
+        Specify the command-line options
+
+        Parameters:
```

Comment:
Should we be documenting all of these options in a place where they are not supported? What is the likelihood that we will remember to to keep this comment up to date when any code changes don't have any effect on this location?